models/User.js

```diff
@@ -0,0 +1,9 @@
+const {model, Schema} = require('mongoose')
+
+const UserSchema = new Schema({
+    email: {type: String, unique: true, required: true}
+})
+
+UserSchema.plugin(require('passport-local-mongoose'))
+
+module.exports = model('user', UserSchema)
```

Comment:
Once[ this PR ](https://github.com/hatchways/team-cocoa-puffs/pull/33)is merged, you can merge it into your branch so we properly sync up on the user model